docs/USAGE.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: documentation
+title: Android openHAB App
+---
+
+{% include base.html %}
+
+# Android openHAB App
+
+The openHAB Android application is a native client for openHAB. The app follows the basic principles of the other UIs, like Basic UI, and presents your predefined openHAB [sitemap(s)}(http://docs.openhab.org/configuration/sitemaps.html).
+
+<a href="https://play.google.com/store/apps/details?id=org.openhab.habdroid"><img src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" height="80"></a>
+
+## Features
+* Control your openHAB server and [openHAB Cloud instance](https://github.com/openhab/openhab-cloud), e.g., an account with [myopenhab](http://www.myopenhab.org/)
+* [Receive notifications via openHAB Cloud](http://docs.openhab.org/addons/actions.html)
+* Change items via NFC tags
+* Send voice commands to openHAB
+* Supports wall mounted tablets
+
+<div class="row">
+  <div class="col s12 m5"><img src="images/main_menu.png" alt="Demo Overview"></div>
+  <div class="col s12 m5"><img src="images/widget_overview.png" alt="Demo Widget Overview"></div>
```

Comment:
My bad. This will look bad as it only occupies 10/12th of the screen width on a PC screen.
Either change both lines to m6 or insert some space in the middle:

```html
  <div class="col s12 m6">
  <div class="col s12 m6">
```

```html
  <div class="col s12 m5">
  <div class="col s12 m5 offset-m2">
```

Reply:
Changed it